controllers/sriovnetworknodepolicy_controller.go

```diff
@@ -152,6 +157,47 @@ func (r *SriovNetworkNodePolicyReconciler) SetupWithManager(mgr ctrl.Manager) er
 		Complete(r)
 }
 
+func (r *SriovNetworkNodePolicyReconciler) syncOvnNetAttachDefs(pl *sriovnetworkv1.SriovNetworkNodePolicyList) error {
+	logger := r.Log.WithName("syncOvnNetAttachDefs")
+	logger.Info("Start to sync default ovn net-attach-defs")
+
+	for _, p := range pl.Items {
+		if p.Name == "default" {
+			continue
+		}
+		if p.Spec.EswitchMode == sriovnetworkv1.ESWITCHMODE_SWITCHDEV {
```

Comment:
I think we should have a flag to disable the render RenderNetAttDef. We see  other use-case for other cni that we can leverage as well.  In the future I think we should create several network types 
1. sriovnetwork
2. ibsriovnetwork
3. ovnkubnetwork
4. ovsnetwork
5. acceleratebridgenetwork

And that should implies the  SR-IOV mode (legacy/switchdev) 
What do you think?

Reply:
@moshe010 sorry for late response.
By disabling RenderNetAttDef, we are expecting user to create their own net-attach-def based on the networktypes they will be using, am I understand it correctly ?

Reply:
yes

Reply:
Could the flag be a env var that passed in to operator.yml or do you think it's prefered to be an API option in SriovOperatorConfig?

Reply:
> I think we should have a flag to disable the render RenderNetAttDef. We see other use-case for other cni that we can leverage as well. In the future I think we should create several network types
> 
>     1. sriovnetwork
> 
>     2. ibsriovnetwork
> 
>     3. ovnkubnetwork
> 
>     4. ovsnetwork
> 
>     5. acceleratebridgenetwork
> 
> 
> And that should implies the SR-IOV mode (legacy/switchdev)
> What do you think?

Would there be cases that one network type may use both SR-IOV modes?